LoopKit/TherapySettings.swift

```diff
@@ -0,0 +1,108 @@
+//
+//  TherapySettings.swift
+//  LoopKit
+//
+//  Created by Anna Quinlan on 7/6/20.
+//  Copyright © 2020 LoopKit Authors. All rights reserved.
+//
+
+import HealthKit
+
+public struct TherapySettings: Equatable {
```

Comment:
I don't quite understand why we need this struct in addition to `StoredSettings`.  There seems to be a strong overlap, with mixed responsibilities for both.  I've added a dev sync topic this week to discuss.